src/components/Popups/FailedNetworkSwitchPopup.tsx

```diff
@@ -12,26 +12,32 @@ const RowNoFlex = styled(AutoRow)`
   flex-wrap: nowrap;
 `
 
+const ColumnContainer = styled(AutoColumn)`
+  margin: 0 12px 0;
```

Comment:
do you just want margin 12px on the right/left here? 
```suggestion
  margin: 0 12px;
```